api/swagger.yaml

```diff
@@ -2339,6 +2363,9 @@ definitions:
   # The Swarm information for `GET /info`. It is the same as `GET /swarm`, but
   # without `JoinTokens`.
   ClusterInfo:
+    description: |
+      ClusterInfo represents info about the cluster for outputting in "info"
```

Comment:
Ugh I wish this had not been called ClusterInfo. We don't refer to them as clusters but as swarms.